src/js_native_api.h

```diff
@@ -566,6 +566,11 @@ NAPI_EXTERN napi_status NAPI_CDECL napi_object_seal(napi_env env,
                                                     napi_value object);
 #endif  // NAPI_VERSION >= 8
 
+#ifdef NAPI_EXPERIMENTAL
+NAPI_EXTERN napi_status NAPI_CDECL napi_can_call_into_js(napi_env env,
```

Comment:
From dicussion in Node-api team meeting today new methods should use `node_api` ->  `node_api_can_call_into_js` in this case.

Overall there was agreement that adding a method makes sense.

The other suggestion was that we need a unit test.